### PR TITLE
Implement base features

### PR DESCRIPTION
**Pull request by denysdovhan**
Base options has been finished. Now `textr-cli` can process text from file or `stdin`, transform one using transformers which depend on `locale`, write result into file or `stdout`, output help.

I think that's enoght for first versions, so after merge this pr, I'm gonna to release `v0.1.0`.

@shuvalov-anton, @iamstarkov, I'm waiting for your review.

**P.S.** `--watch` and `--diff` features will be implemented as soon as possible.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -2,3 +2,4 @@
 node_modules
 npm-debug.log
 *.es5.js
+test/actual
```
Comment (iamstarkov) [inline .gitignore:5]: why its like this?

File: .travis.yml [hunk 2]
```diff
@@ -0,0 +1,8 @@
+language: node_js
+node_js:
+  - '4.1'
+  - '4.0'
+  - '0.12'
```
Comment (iamstarkov) [inline .travis.yml]: take a look at https://github.com/iamstarkov/generator-travis

File: cli/index.js [hunk 3]
```diff
@@ -0,0 +1,105 @@
...
+ * Render text string throught textr transformer
+ * @param  {String} text Text that will be processed
+ * @param  {Array}  tfs  Array of transformers
+ * @return {Void}        Write into strout or file and exit
+ */
+const render = (text, tfs = [], locale = 'en-us') => {
+  const res = (textr({ locale }).use.apply(null, tfs))(text);
+  if (cli.flags.outFile) {
+    try {
+      fs.writeFileSync(cwd(cli.flags.outFile), res, 'utf8');
```
Comment (iamstarkov) [inline cli/index.js:64]: are you sure about using `*Sync` methods in NodeJs?

**Comment by iamstarkov** [comment]
LGTM

File: .gitignore (hunk 1 above)
Comment (denysdovhan) [inline .gitignore:5]: That's for testing `-o` flag. It should write file.

File: .travis.yml (hunk 2 above)
Comment (denysdovhan) [inline .travis.yml]: OK, I'll fix this.

File: cli/index.js (hunk 3 above)
Comment (denysdovhan) [inline cli/index.js:64]: In cli apps better to use blocking io or use promises. Shell scripts exec from top to bottom, so async seem to be inconvenient.
https://strongloop.com/strongblog/node-js-v0-12-shell-programming-synchronous-child-process/

File: .gitignore (hunk 1 above)
Comment (iamstarkov) [inline .gitignore:5]: okie-dokie

File: cli/index.js (hunk 3 above)
Comment (iamstarkov) [inline cli/index.js:64]: okay

**Comment by iamstarkov** [comment]
👍

**Comment by A** [comment]
Super! Thank you, @denysdovhan :D

File: .travis.yml (hunk 2 above)
Comment (denysdovhan) [inline .travis.yml]: I don't know why, but using `v5` and `v4` break executing of `before_script`.

Comment (denysdovhan) [inline .travis.yml]: Fixed this thanks for `before_install` hook.

Comment (iamstarkov) [inline .travis.yml]: i didnt understand anything

**Pull request merged**